src/operators.ts

```diff
@@ -75,6 +75,21 @@ const operators: {
     }
     return v < compare;
   },
+  $exist: (v, compare) => {
+    if (
+      !supportedType(v, ['number', 'string', 'boolean', 'null', 'undefined']) ||
+      !supportedType(compare, ['boolean'])
+    ) {
+      throw new Error('Unsupported types for $exist operator');
+    }
```

Comment:
Shouldn't exist essentially support any type? Ultimately we are just looking at if the key exist so the type shouldn't matter!

Reply:
Right!